ports/veigar/portfile.cmake

```
@@ -0,0 +1,27 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO winsoft666/veigar
+    HEAD_REF master
+    REF ${VERSION}
+    SHA512 9a967c629f63b96327638e8a653b22f5e7c754d4d92b6d75eb8c5f6256d7f413ad15c32163dfe1b6abba611ba9745139ce979e4018ee5c6a80a9cc1b08715648
+)
+
+string(COMPARE EQUAL "${VCPKG_CRT_LINKAGE}" "static" VEIGAR_USE_STATIC_CRT)
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+    OPTIONS
+        -DVEIGAR_USE_STATIC_CRT:BOOL=${VEIGAR_USE_STATIC_CRT}
+        -DVEIGAR_BUILD_TESTS:BOOL=OFF
+        -DVEIGAR_BUILD_EXAMPLES:BOOL=OFF
+)
+
+vcpkg_cmake_install()
+
+vcpkg_cmake_config_fixup(CONFIG_PATH share/veigar)

```

```suggestion
vcpkg_cmake_config_fixup()
```
Default is `share/${PORT}`, so we don't need to be explicit.